src/main/java/core/basesyntax/FileWork.java

```
@@ -8,7 +13,31 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final String START_LETTER = "w";
+    
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder result = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            String readLine;
+            while ((readLine = reader.readLine()) != null) {
+                String[] bufferArray = readLine.split(" ");
+                for (String line : bufferArray) {
+                    if (line.toLowerCase().startsWith(START_LETTER)) {
+                        result.append(line.toLowerCase().replaceAll("\\W", "")
+                                + " ");
+                    }
+                }
+            }
+            String[] resultArray = result.toString().split(" ");
+            if (resultArray.length == 1 && resultArray[0].equals("")) {
+                return new String[0];
+            }
+            Arrays.sort(resultArray);
+            reader.close();
+            return resultArray;
+        } catch (IOException e) {
+            e.getStackTrace();
```

don't ignore exception, re-throw them